Jenkinsfile

```diff
@@ -84,10 +84,22 @@ node {
         ],
     ]])
     try_wrapper(mail_failure) {
+        deleteDir()
+        def image = null
+        stage('builder image') {
+            dir('aos-cd-jobs') {
+                checkout scm
+                image = docker.build 'ose-builder', 'builder'
```

Comment:
The plugin allows local builds, so we don't even need to publish the image.
The `ose-builder` tag here is arbitrary, since we get back a reference to the
image.